dashboard/src/app/ide/ide.html

```diff
@@ -15,12 +15,13 @@
   <div class="ide-page-loader"
        layout="column" layout-align="center center">
     <div ng-hide="ideCtrl.selectedWorkspace">No workspace selected. Unable to open IDE</div>
-    <div layout="column" class="ide-page-loader-content" ng-show="ideCtrl.selectedWorkspace">
-      <div class="spinner">
-        <div class="rect1"></div>
-        <div class="rect2"></div>
-        <div class="rect3"></div>
-      </div>
+    <div class="ide-page-loader-content" ng-show="ideCtrl.selectedWorkspace">
+        <div>
+          <span>Loader idea</span>
```

Comment:
Could you attach an animated gif ?
Because I'm not sure we should display "loader idea" text